meteor/client/ui/Settings/components/ConfigManifestEntryComponent.tsx

```diff
@@ -75,6 +84,7 @@ export const ConfigManifestEntryComponent = withTranslation()(
 					<label className="field">
 						{t(configField.name)}
```

Comment:
Is there any localization effort included here, to provide translations for these (i.e. the manifest provides it's own locale tables)? If not, then - since these will now be inserted from the manifest, I don't think there's a point in passing these through the localization engine. There's no chance of any of these strings ending up inside the Core localization.

Reply:
Yeap, agreed and removed the calls to i18n